Sources/Views/Cells/MessageContentCell.swift

```diff
@@ -170,9 +173,8 @@ open class MessageContentCell: MessageCollectionViewCell {
         messageBottomLabel.attributedText = bottomMessageLabelText
         timeLabel.attributedText = timeLabelText
 
-        let displayTimeBySwipingLeft = messagesCollectionView.clipsToBounds == false
-        messageBottomLabel.isHidden = displayTimeBySwipingLeft
-        timeLabel.isHidden = !displayTimeBySwipingLeft
+        messageBottomLabel.isHidden = messagesCollectionView.displayTimeBySwipingLeft
```

Comment:
just curious, why are we hiding this?

Reply:
Because this label shows time and date.

Reply:
AFAIK this label can display any text set by a client, there is no need to hide it while displaying message timestamp